src/main/kotlin/racingcarWinner/ui/OutputView.kt

```diff
@@ -1,38 +1,25 @@
 package racingcarWinner.ui
 
-import racingcarWinner.core.Car
-import racingcarWinner.core.MessageCode
-import racingcarWinner.core.Winner
-import racingcarWinner.core.WinnerRacing
+import racingcarWinner.domain.Car
+import racingcarWinner.util.MessageCode
 
 object OutputView {
     private const val MOVE_STEP = "-"
-    private const val SEPARATOR = " "
     private const val COMMA = ", "
-    private const val COLON = " : "
-    private const val ESCAPE_LINE = "\n"
-    private const val ESCAPE_DOUBLE_LINE = "\n\n"
+    private const val COLON = ":"
 
-    fun outputResult(cars: List<Car>, winner: Winner, tryNumber: Int) {
+    fun outputResult() {
         println(MessageCode.Result.message)
-        print(tryAllRacingOutput(cars, winner, tryNumber))
     }
 
-    fun outputWinner(cars: List<Car>, winner: Winner) {
-        val winnerList = winner.getWinner(cars).joinToString(COMMA)
-        println(ESCAPE_LINE.plus(winnerList).plus(MessageCode.WinnerResult.message))
-    }
-
-    private fun toMoveStepOutput(moveStep: Int): String =
-        (WinnerRacing.INIT_TRY_NUMBER..moveStep).joinToString(SEPARATOR) { MOVE_STEP }
-
-    private fun tryOneRacingOutput(cars: List<Car>, winner: Winner, tryNumber: Int): String =
-        WinnerRacing.startRacing(cars, winner, tryNumber).joinToString(ESCAPE_LINE) { car ->
-            car.carName.plus(COLON).plus(toMoveStepOutput(car.moveStep))
+    fun printTryOneRacingOutput(racingCars: List<Car>) {
+        racingCars.forEach {
+            println("${it.carName} $COLON ${MOVE_STEP.repeat(it.getMoveStep())}")
         }
+        println()
+    }
 
-    private fun tryAllRacingOutput(cars: List<Car>, winner: Winner, tryNumber: Int): String =
-        (WinnerRacing.INIT_TRY_NUMBER..tryNumber).joinToString(ESCAPE_DOUBLE_LINE) {
-            tryOneRacingOutput(cars, winner, it)
-        }
+    fun outputWinner(winnerList: List<String>) {
+        println(winnerList.joinToString(COMMA).plus(MessageCode.WinnerResult.message))
```

Comment:
```suggestion
        println("${winnerList.joinToString(",")} ${MessageCode.WinnerResult.message}")
```

이렇게 해주면 `plus`를 사용하지 않아도 될 것 같습니다.